swagger.yaml

```diff
@@ -5415,6 +5425,19 @@ definitions:
       console_url:
         type: string
 
+  host-label-params:
+    type: object
+    required:
+      - 'label_name'
+      - 'label_set'
+    properties:
+      label_name:
```

Comment:
Is this the label key?  If so, let's call it label_key.  Why not also add label_value?

Reply:
I thought label_name and label_set would be better suited here since label_name suggests that this is the name of the label and label_set would be a boolean suggesting whether to set or unset the label.

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
Kubernetes labels have a key and value

Reply:
As I mentioned in another comment, label_set is not used in this PR, and I don't think it's needed - just overwrite the list of labels on each update.

Reply:
Understood